src/TryMudBlazor.Client/Pages/Repl.razor.cs

```diff
@@ -37,9 +37,6 @@ public partial class Repl : IDisposable
         [Inject]
         public IJSInProcessRuntime JsRuntime { get; set; }
 
-        [Inject]
-        public IJSUnmarshalledRuntime UnmarshalledJsRuntime { get; set; }
```

Comment:
`IJSUnmarshalledRuntime` was removed completely, and it wasn't actually used in the code anymore anyway.